src/shared/components/User/Avatar.tsx

```diff
@@ -7,7 +7,12 @@ export interface AvatarProps {
 
 const UserAvatar: React.FunctionComponent<AvatarProps> = props => {
   const { createdBy } = props;
-  const [name] = createdBy.split('/').slice(-1);
+  let name = createdBy;
+  try {
+    [name] = createdBy.split('/').slice(-1);
```

Comment:
same

Reply:
added tests in 759c033fceb7fcf8ad141fea41c50e61ee195945